lib/extensions/base.py

```diff
@@ -164,9 +165,9 @@ def descendants(self, node, selected=False, troubleshoot=False):  # noqa: C901
             nodes.extend(self.descendants(child, selected, troubleshoot))
 
         if selected:
-            if node.tag in EMBROIDERABLE_TAGS or is_embroiderable_clone(node):
+            if (node.tag in EMBROIDERABLE_TAGS or is_embroiderable_clone(node)) and not (node.tag == SVG_PATH_TAG and not node.get('d', '')):
```

Comment:
Why do we need to specially handle paths with empty D here?

I was thinking we could just let them get added and then they'll produce an empty patch list because of the `to_patches()` above... does that not work?

Reply:
This would cause problems if no other objects were selected if I remember right. And we would need to catch it for all the different element types. I thought, it'd be easier to catch it upfront.